src-docs/spec/sendUpdate.md

```diff
@@ -17,6 +17,9 @@ Send an update to all peers.
        Do not add linebreaks; implementations will truncate the text at about 50 characters or less.
        If there are series of info messages, older ones may be dropped.
        use this option sparingly to not spam the chat.
+    - `update.notify`: optional array of [user addresses](./selfAddr_and_selfName.md)
```

Comment:
to increase the usefulness of this API, would be nice if dev could pass `update.notify=["all"]` to avoid having to collect user IDs before being able to notify, for example a calendar app, it is good that events cause notification without the need of every user opening the app first